src/workerd/io/actor-sqlite.c++

```diff
@@ -198,6 +198,7 @@ void ActorSqlite::ExplicitTxn::rollbackImpl() noexcept(false) {
 
 void ActorSqlite::onWrite() {
   if (currentTxn.is<NoTxn>()) {
+    requireNotBroken();
```

Comment:
Let's move this above the `if()`, so that all writes will fail when broken.

Reply:
I moved the line, and also added a rudimentary test.